@ecomplus/storefront-components/src/js/ProductCard.js

```diff
@@ -92,6 +92,10 @@ export default {
     i19addToFavorites: () => i18n(i19addToFavorites),
     i19outOfStock: () => i18n(i19outOfStock),
     i19unavailable: () => i18n(i19unavailable),
+    i19addToFavorites: () => i18n({
+      pt_br: 'Adicionar aos favoritos',
+      en_us: 'Add to favorites'
+    }),
```

Comment:
você tá revertendo uns fixes que eu dei, provavelmente seu master estava desatualizado 😬 
sempre que for começar uma PR nova tem que dar pull primeiro, depois criar um novo branch do master